index.js

```diff
@@ -1,102 +1,53 @@
 const fs = require('fs')
-
-const OVERRIDE_DIR = 'overrides'
-const GENERATED_DIR = 'generated'
-const LISTS_DIR = 'features'
+const utils = require('./lib/utils')
```

Comment:
This brings in a whole heap of require/export can we move to ESM like content-scope-scripts does? It should be just a case of adding _"type": "module"_ to the package.json as we're just a node script.